src/app/loadout-builder/filter/ModPicker.tsx

```diff
@@ -50,6 +50,14 @@ const sortMods = chainComparator<LockedArmor2Mod>(
   compareBy((l) => l.modDef.displayProperties.name)
 );
 
+function isGeneralOrOtherOrRaid(category: ModPickerCategory) {
```

Comment:
Unfortunately this function name tells me what it *does* but not what it's *for*. What are we trying to represent with this, and why does it imply you can select 5 mods vs 2?

Reply:
Yeah I know exactly what you mean and it bothered me as well. Problem is it serves two complete seperate purposes. Whether we allow 2 or 5 total mods to be selected OR whether we need to care about mod energies matching. I would just use the condition directly but its pretty large to put into a ternary imo. Let me know what you want to do with it and happy to go that route. I'm not happy with most things I have come up with.

Reply:
In order to unblock this PR and address the comment, for now can this be renamed to "isSlotSpecificCategory" or something that states its purpose?